src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -179,7 +179,12 @@ public void load(CoreExtension.Context context) {
                             .name("Repository Slug for the Gitlab (Server or Cloud) instance")
                             .description("The repository slug can be either in the form of user/repo or it can be the Project ID")
                             .type(PropertyType.STRING)
-                            .build()
+                            .build(),
+                            
+                    PropertyDefinition.builder(GitlabServerPullRequestDecorator.PULLREQUEST_COMMENTS_MIN_SEVERITY).category(PULL_REQUEST_CATEGORY_LABEL).subCategory(GITLAB_INTEGRATION_SUBCATEGORY_LABEL)
+                            .onQualifiers(Qualifiers.PROJECT).name("Min Comment Severity")
+                            .type(PropertyType.SINGLE_SELECT_LIST).options("BLOCKER", "CRITICAL", "MAJOR", "MINOR", "INFO").build()
```

Comment:
Instead of hardcoding the severity list you could just build the list e.g. like
```
Arrays.stream(Severity.values())
      .map(Severity::name)
      .collect(Collectors.toList())
```

Reply:
Oh, I didn't find the enum from Sonar and didn't want to create an enum myself which could be difficult to keep in sync. I will change it to your suggestion.